src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -204,7 +204,7 @@ private bool IsNotAbsoluteUri
         //
         // Statically checks if Iri parsing is allowed by the syntax & by config
         //
-        internal static bool IriParsingStatic(UriParser syntax)
+        internal static bool IriParsingStatic(UriParser? syntax)
```

Comment:
`UriParser` in `AllowIdnStatic` should also be nullable

Reply:
Fixed